### PR TITLE
refactor: code refactor. + updated the  documentation

### PR DESCRIPTION
**Pull request by IgnatG**
### Changelog
- updated the types from `string` to `Dates` (where applicable)
- updated the documentation
- updated the NPM packages

### CODE REVIEW

File: src/modules/currencies/entities/currency.entity.ts [hunk 1]
```diff
@@ -22,24 +23,24 @@ export class CurrencyEntity {
   isoCode: string;
 
   @Column({ name: 'DATE_CREATED_DATETIME', type: 'timestamp' })
-  @ApiProperty({ example: '2010-03-15 16:13:34.000' })
+  @ApiProperty({ example: '2017-03-06T16:20:17' })
   @Transform(({ value }) => format(value, "yyyy-MM-dd'T'HH:mm:ss"))
```
Comment (avaitonis) [inline src/modules/currencies/entities/currency.entity.ts:27]: Should we remove @Transform here and use new date format "2021-01-26T23:59:59.000Z"

File: src/modules/sector-industries/entities/sector-industry.entity.ts [hunk 2]
```diff
@@ -87,15 +87,15 @@ export class SectorIndustryEntity {
   @ApiProperty({ example: '2010-03-15 00:00:00.000' })
   created: Date;
 
-  @Column({ name: 'DATE_LAST_UPDATED_DATETIME' })
-  @ApiProperty({ example: 0, description: '2010-03-15 00:00:00.000' })
+  @Column({ name: 'DATE_LAST_UPDATED_DATETIME', type: 'timestamp' })
+  @ApiProperty({ example: '2010-03-15 00:00:00.000' })
```
Comment (avaitonis) [inline src/modules/sector-industries/entities/sector-industry.entity.ts:91]: Result is in format: 2017-04-01T00:00:00.000Z
Example could be in same format.

Comment (IgnatG) [reply src/modules/sector-industries/entities/sector-industry.entity.ts]: good shout - updated

File: src/modules/currencies/entities/currency.entity.ts (hunk 1 above)
Comment (IgnatG) [reply src/modules/currencies/entities/currency.entity.ts]: good shout - updated

**Pull request merged**